src/documents/APIChangesFrom1_5To2_0.md

```diff
@@ -10,6 +10,59 @@ This page contains a list of the features, APIs, and etc. that have changed in I
 
 ### Preferences Framework ###
 
+#### Panel Changes ####
+Changes and additions are being made to UI Options in order to improve its compliance with accesibility standards.
+
+##### Message Bundle Keys #####
+
+######Additions######
+* `textSizeDescr: {messagekey: "textSizeDescr"}`
+* `textFontDescr: {messagekey: "textFontDescr"}`
+* `lineSpaceDescr: {messagekey: "lineSpaceDescr"}`
+* `contrastDescr: {messagekey: "contrastDescr"}`
+
+######Changes######
+<table>
+<tr><th> 1.5</th><th>2.0</th></tr>
+<tr>
+    <td><code>choiceLabel: {messagekey: "speakChoiceLabel"}</code></td>
+    <td><code>speakDescr: {messagekey: "speakDescr"}</code></td>
+</tr>
+<tr>
+    <td><code>choiceLabel: {messagekey: "tocChoiceLabel"}</code></td>
+    <td><code>tocDescr: {messagekey: "tocDescr"}</code></td>
+</tr>
+<tr>
+    <td><code>choiceLabel: {messagekey: "simplifyChoiceLabel"}</code></td>
+    <td><code>simplifyDescr: {messagekey: "simplifyDescr"}</code></td>
+</tr>
+</table>
+
+##### Selectors #####
+
+######Additions######
+* `contrastDescr: .flc-prefsEditor-contrast-descr`
+* `lineSpaceDescr: .flc-prefsEditor-line-space-descr`
+* `textSizeDescr: .flc-prefsEditor-text-size-descr`
+* `textFontDescr: .flc-prefsEditor-text-font-descr`
+
+######Changes######
+<table>
+<tr><th> 1.5</th><th>2.0</th></tr>
+<tr>
+    <td><code>choiceLabel: ".flc-prefsEditor-toc-choice-label"</code></td>
+    <td><code>tocDescr: ".flc-prefsEditor-toc-descr"</code></td>
+</tr>
+<tr>
+    <td><code>choiceLabel: ".flc-prefsEditor-speak-choice-label"</code></td>
+    <td><code>speakDescr: ".flc-prefsEditor-speak-descr"</code></td>
+</tr>
+<tr>
+    <td><code>choiceLabel: ".demo-prefsEditor-simplify-choice-label"</code></td>
+    <td><code>simplifyDescr: ".demo-prefsEditor-simplify-descr"</code></td>
```

Comment:
The "demo" ones don't need to be included in this document. They aren't actually part of any supported API.